File: MusicPlayer.v2/app/src/main/java/com/example/musicplayerv2/Callback.java
package com.example.musicplayerv2;

import android.content.SharedPreferences;

/**
 * Created by ${Aina} on 11.11.2018.
 */
public interface Callback {
    void callback(int position);

    void onSharedPreferenceChanged(SharedPreferences sharedPreferences, String key);
}

File: MusicPlayer.v2Kotlin/app/src/main/java/com/example/musicplayerv2/Callback.kt
package com.example.musicplayerv2

import android.content.SharedPreferences

/**
 * Created by ${Aina} on 11.11.2018.
 */
interface Callback {
    fun exactSong(position: Int)
    fun onSharedPreferenceChanged(sharedPreferences: SharedPreferences, key: String)
}
